src/main/java/com/google/swarm/tokenization/common/DLPTransform.java

```diff
@@ -109,7 +109,7 @@ public void processElement(ProcessContext c) throws IOException {
             long bytesInspected = contentItem.getSerializedSize();
             int totalFinding =
                 Long.valueOf(response.getResult().getFindingsList().stream().count()).intValue();
-            LOG.debug("bytes inspected {}", bytesInspected);
+            LOG.debug("DLPTransform:DLPInspect: Bytes inspected {}", bytesInspected);
```

Comment:
Contents are not inspected yet. So its going to be misleading. I woulkd take it out

Reply:
That's right. Will take it out.